gapic-generator/lib/google/gapic/schema/api.rb

```diff
@@ -64,6 +64,12 @@ def file_for address
           matching_files.first
         end
 
+        def fix_inflection str
+          str = String str
+          return str if configuration[:inflections].nil?
+          configuration[:inflections].fetch str, str
```

Comment:
Right now custom inflections are applied specifically to file paths and namespaces. Do we know if that's the full list? (And will there be cases where different inflections would need to be applied to the two? I'm guessing not because one is always capitalized whereas the other is never capitalized.) I know we can't predict for certain, but want to think about this a bit and try to avoid too many breaking changes in the future.

Reply:
The only place we are using inflections now is calculating the namespace, calculating the file path, calculating a method name, and calculating the stub name (`speech_stub`, `operations_stub`, etc.). We could call `fix_inflection` on method and stub names if it was warranted. Or, we could maintain separate list of inflections for namespaces, file paths, method names, and stub names.

Reply:
This has been updated to add separate configs for file path and namespace.